README.md

```diff
@@ -5,9 +5,10 @@
 
 ## Author 
 - [Arthur Zucker](https://github.com/ArthurZucker)
+- [Chris Rauch](https://github.com/chrisrauch193)
 
 ## Installation 
-Clone this repository 
+Clone this repository
 
```

Comment:
Let's also add that it should no be possible to just run 
`pip install git+https://github.com/ArthurZucker/PAMAI`